repositories.yaml

```diff
@@ -282,9 +283,11 @@ android-places:
   app_id: android-places
   notification_emails:
     - frank@mozilla.com
+    - sync-team@mozilla.com
```

Comment:
And took the liberty of sharing the notification email load with poor frank, since the sync team is ultimately responsible for these metrics :-)